src/components/SearchCard/SearchCardList.tsx

```diff
@@ -2,11 +2,18 @@ import styles from './SearchCardList.module.css';
 import { SearchCardProps } from '../../lib/types';
 import InCollection from '../../assets/InCollection';
 
-export default function SearchCard({ searchResults }: SearchCardProps) {
+export default function SearchCard({
```

Comment:
You renamed the file but not the component :P 

```suggestion
export default function SearchCardList({
```